src/ExpandObjects/epfilter.f90

```diff
@@ -23691,6 +23697,7 @@ SUBROUTINE CreateFanCoil
     CALL AddToObjFld('Fan Efficiency', base + fczSupplyEfficiencyOff,'')
     CALL AddToObjFld('Pressure Rise {Pa}', base + fczSupplyPressureOff,'')
     CALL AddToObjStr('Maximum Flow Rate {m3/s}','autosize')
+	if ( capacityControlKind .EQ. capctrlASHRAE90VariableFan ) CALL AddToObjStr('Low Speed Supply Air Flow Ratio','autosize')
```

Comment:
This is also wrong, and should have been in the fan coil unit not the fan. Again, won't hurt anything since capacityControlKind can never be capctrlASHRAE90VariableFan since that key is not in the idd. I'll correct this when I get a chance.

Reply:
OK on waiting with HVACTemplate support for the new control option.  Be sure to start a new branch/PR for that.